Description: Move glib-2.0 Requires in upstream enchant.pc.in to Requires.private. While enchant depends on glib-2.0, 
API it exposes to other programs doesn't require glib. This makes life easier for packages depending on enchant
Author: Agustin Martin Domingo <agmartin@debian.org>
Index: enchant/enchant.pc.in
===================================================================
--- enchant.orig/enchant.pc.in	2011-12-14 15:36:23.000000000 +0100
+++ enchant/enchant.pc.in	2011-12-20 17:11:31.434860468 +0100
@@ -6,6 +6,6 @@
 Name: libenchant
 Description: A spell checking library
 Version: @VERSION@
-Requires: glib-2.0 gmodule-no-export-2.0
+Requires.private: glib-2.0 gmodule-no-export-2.0
 Libs: -L${libdir} -lenchant
 Cflags: -I${includedir}/enchant
